src/main/java/uk/gov/hmcts/juror/api/moj/utils/JurorResponseUtils.java

```diff
@@ -61,6 +69,22 @@ public static PaperResponse createMinimalPaperSummonsRecord(Juror juror,
         return jurorPaperResponse;
     }
 
+    public static void updateCurrentOwnerInResponseDto(JurorPoolRepository jurorPoolRepository,
+                                                       IJurorResponse responseDto) {
+
+        // set the current owner.  Need to ensure the current owner is returned as the owner can change if, for
+        // example, the juror is transferred to a different pool
+        List<JurorPool> jurorPools =
+            JurorPoolUtils.getActiveJurorPoolRecords(jurorPoolRepository, responseDto.getJurorNumber());
+
+        Optional<JurorPool> jurorPool = jurorPools.stream()
+            .filter(not(jp -> jp.getStatus().getCode().equals(IJurorStatus.TRANSFERRED)))
+            .sorted(Comparator.comparing(JurorPool::getDateCreated).reversed())
```

Comment:
Would be better to filter and sort in DB then only return one record to prevent Java filtering